.github/workflows/pull-request.yml

```
@@ -257,6 +257,120 @@
+        run: cp .env.example .env
+      
+      - name: Update API_URL in .env for integration tests
+        run: echo "API_URL=http://127.0.0.1:4000/graphql" > .env
+      
+      - name: Install Chrome for Web Testing
+        run: |
+          wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+          sudo sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list'
+          sudo apt-get update
+          sudo apt-get install -y google-chrome-stable
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)